### ISSUE TITLE
Missing proper keyboard bindings

### ISSUE THREAD

**Issue by WW3** [sagalbot/vue-select#943]
The keyboard operability doesn't align with native select. For this component to be accessible and comply with WCAG, please consider this.
Also, instead of role="combobox" change to "listbox" as this component doesn't include an editable search filed which is characteristic of a Combobox

**Comment by sagalbot** [comment]
Which keyboard inputs am I missing with native select?

https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html

I've read through the aria spec a few times now, and my understanding is that the search input is a combobox, and the dropdown is a listbox. If I'm incorrect here, can you explain why?

**Comment by FabianEllenberger** [comment]
I think the `aria-selected` on the `li` tags are missing, in order for screen readers to read the current selected item when navigation through the list with the keyboard (up/down keys).

Awesome component btw.! 🙏
